EIPS/eip-3664.md

```diff
@@ -0,0 +1,383 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends ERC-1155.
```

Comment:
```suggestion
description: Extend Non-Fungible Tokens with additional metadata for the metaverse
```

I think it can be inferred that this EIP is a standard, and "next generation" doesn't convey anything specific.

Reply:
```suggestion
description: Extend EIP-1155 Non-Fungible Tokens with additional metadata for the metaverse
```

@SamWilsn adding mentioning of EIP-1155 because this EIP seems only touch EIP-1155 not EIP-721 or other ERCs of NFTs.